docs/core-concepts/workflow.md

```diff
@@ -164,26 +164,34 @@ The inference type and ground truth type for a workflow will often look very sim
 
 [Annotation][kolena.workflow.annotation.Annotation] types can be extended to include additional fields, when necessary.
 
-Consider the example of a [`Keypoints`][kolena.workflow.annotation.Keypoints] detection model that detects anywhere from
-0 to N keypoints arrays when provided an image. Each keypoints array has an associated class label and confidence value.
-This model's inference type could be defined as follows:
+Consider the example of a detection model that detects labeled bounding boxes when provided an image.
+The [ScoredLabeledBoundingBox][kolena.workflow.annotation.ScoredLabeledBoundingBox] represents a bounding box with a label and a confidence score.
+If you wanted to add fields found in COCO format annotations (area, isCrowd) then this model's inference type could be defined as follows:
 
 ```python
 from dataclasses import dataclass
 from typing import List
 
 from kolena.workflow import Inference
-from kolena.workflow.annotation import Keypoints
+from kolena.workflow.annotation import ScoredLabeledBoundingBox
 
 @dataclass(frozen=True)
-class ScoredLabeledKeypoints(Keypoints):
-    # points: List[Tuple[float, float]]  # inherited from Keypoints
-    score: float  # confidence score, between 0 and 1
-    label: str  # predicted class
+class CocoScoredLabeledBoundingBox(ScoredLabeledBoundingBox):
```

Comment:
@y27choi I am torn here on using the base BoundingBox to extend compared to the `ScoredLabeledBoundingBox`. The base BoundingBox is simpler, but ScoredLabeledBoundingBox shows a more complete picture. Thoughts?

Reply:
I wonder if we want to update this. The existing keypoint example shows how to extend an annotation type and I don't see a reason to change this.

Reply:
I reverted that change, to stick with the original example